docs/README.md

```diff
@@ -0,0 +1,5 @@
+## Home
```

Comment:
we can delete this file all together

Reply:
@kenzieschmoll then should the docs/ directory be deleted then? This is the only thing in it now.
When we discussed the file structure in [flutter.dev/go/migrate-flutter-wiki-spreadsheet](https://docs.google.com/spreadsheets/d/1x65189ZBdNiLRygpUYoU08pwvXD4M-Z157c6pm8deGI/edit?usp=sharing) , it sounded like you all would self-organize other docs after the wiki was migrated?

Reply:
Either way is fine. I'm happy to move things around later if you want to just land this as is 👍